xbmc/addons/Skin.cpp

```diff
@@ -369,6 +369,9 @@ const INFO::CSkinVariableString* CSkinInfo::CreateSkinVariable(const std::string
 
 void CSkinInfo::OnPreInstall()
 {
+  bool skinLoaded = g_SkinInfo != nullptr;
```

Comment:
And as mentioned in #10990 can we please add a comment here that says what the UnloadSkin is for? Because it makes no sense unless you already know it's a workaround for closing TextureBundle on windows.